pkg/cmd/codespace/ports.go

```diff
@@ -219,7 +219,7 @@ func newPortsVisibilityCmd(app *App, selector *CodespaceSelector) *cobra.Command
 	return &cobra.Command{
 		Use:     "visibility <port>:{public|private|org}...",
 		Short:   "Change the visibility of the forwarded port",
-		Example: "gh codespace ports visibility 80:org 3000:private 8000:public",
+		Example: "$ gh codespace ports visibility 80:org 3000:private 8000:public",
```

Comment:
Wow, okay little bit of a rabbit hole approach to subcommandset.

Again, would love seeing this expanded into long form with a comment:

```suggestion
		Example: heredocs.Doc(`
			# Open HTTP(80) traffic to org members and HTTP(8000) traffic to public, make HTTP(3000) private
			$ gh codespace ports visibility 80:org 3000:private 8000:public",
		`),
```